src/os/bluestore/BlueStore.cc

```diff
@@ -5407,6 +5407,10 @@ void BlueStore::_txc_add_transaction(TransContext *txc, Transaction *t)
       }
       break;
 
+    case Transaction::OP_MERGE_DELETE:
+      assert(0 == "not implemented");
```

Comment:
Nope, we need it implemented.  We *might* make an exception for bluestore only if you talk to the people currently working on it and they don't think now is a good time.